src/main/java/org/snomed/aag/data/services/ProjectAcceptanceCriteriaService.java

```diff
@@ -22,7 +24,7 @@
 
 @Service
 public class ProjectAcceptanceCriteriaService {
-
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProjectAcceptanceCriteriaService.class);
```

Comment:
FYI this is briefer, services are singletons so there will never be more than one instance:
```
private final Logger logger = LoggerFactory.getLogger(getClass());
```
Just sharing an idea, no need to change it. Maybe this is just coding style.

Reply:
Thanks for sharing, it's a good idea. It's similar in nature to excessively using the `final` keyword for variables.